apps/managedidentity/example_test.go

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package managedidentity_test
+
+import (
+	"context"
+	"fmt"
+
+	mi "github.com/AzureAD/microsoft-authentication-library-for-go/apps/managedidentity"
+)
+
+var miSystemassignedClient mi.Client
+var err error
+
+func ExampleNew() {
+	// System assigned Client
+	miSystemassignedClient, err = mi.New(mi.SystemAssigned())
+	if err != nil {
+		// TODO: Handle error
+	}
+	_ = miSystemassignedClient
+
+	// User assigned Client
+	clientId := "ClientId" // TODO: replace the correct Client id
+
+	miClientIdAssignedClient, err := mi.New(mi.UserAssignedClientID(clientId))
+	if err != nil {
+		// TODO: Handle error
+	}
+	_ = miClientIdAssignedClient
+
+	// Output: -
```

Comment:
Suggest removing this because it doesn't improve the example--there's no output to help readers understand how the API works (to be clear, this example doesn't need any)--or test coverage (note that `go test` always compiles examples, so `// Output` isn't required; it just makes `go test` execute an example)